shell/platform/android/android_choreographer.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_ANDROID_ANDROID_CHOREOGRAPHER_H_
+#define FLUTTER_SHELL_PLATFORM_ANDROID_ANDROID_CHOREOGRAPHER_H_
+
+#include "flutter/fml/macros.h"
+
+#include <cstdint>
+
+namespace flutter {
+
+//------------------------------------------------------------------------------
+/// The Android Choreographer is used by `VsyncWaiterAndroid` to await vsync
+/// signal. It's only avialalbe on API 29+ or API 24+ and architecture is 64-bit
```

Comment:
```suggestion
/// signal. It's only available on API 24+.
```

API 29 added support for an explicit 64-bit int (int64_t).

Reply:
done